st2common/st2common/openapi.yaml

```diff
@@ -4760,6 +4760,8 @@ definitions:
     type: object
   WebhookBody:
     description: Given support of Content-Type header, that could be anything.
+    x-api-model: st2common.models.api.webhook:WebhookBodyAPI
+    x-api-use-data-arg: True
```

Comment:
I'm open to a better name of course.

It basically signals the router, that this API endpoint uses a special API model object where the actual input body / data is stored in the model `data` attribute.

With that, we can support arbitrary types for input data which are not an object.